### PR TITLE
Fixed click on field on map redirection issue

### PR DESCRIPTION
**Pull request by KennethWrong**
Added a event listener to the GeoJson component in index.svelte to redirect client to view the field dashboard.

### CODE REVIEW

File: src/routes/index.svelte [hunk 1]
```diff
@@ -2,6 +2,12 @@
   import { Field } from '$lib/db';
   import Leaflet from '$lib/leaflet/Leaflet.svelte';
   import GeoJson from '$lib/leaflet/GeoJson.svelte';
+  import { goto } from '$app/navigation';
+
+
+const handle_click_event = (field : Field) => {
```
Comment (abalmos) [inline src/routes/index.svelte:8]: Can you make the name more meaningful, maybe `goToField` ?

Please use an actual function rather then anonymous one.

File: package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
     "postcss-load-config": "^3.1.0",
     "prettier": "~2.4.1",
     "prettier-plugin-svelte": "^2.2.0",
-    "svelte": "^3.42.6",
+    "svelte": "^3.44.0",
```
Comment (abalmos) [inline package.json:35]: Was there a reason to update?

We need to test everything well if we do.

File: src/lib/components/FieldCard.svelte [hunk 3]
```diff
@@ -92,9 +92,10 @@
 </script>
 
 <div class="card rounded-none">
-  <figure on:click={() => goto(`/fields/${encodeURIComponent(field.id)}`)}>
+    <figure on:click={()=>goto(`/fields/${encodeURIComponent(field.id)}`)}>
```
Comment (abalmos) [inline src/lib/components/FieldCard.svelte:95]: Looks like you need to get prettier into your editor or run it on everything before committing.

File: yarn.lock [hunk 4]
```diff
@@ -3,83 +3,83 @@
 
 
 "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.14.5":
-  version "7.15.8"
```
Comment (abalmos) [inline yarn.lock:6]: Do you have any idea why every line in this file was changed? With SvelteKit being so new, untested updates have been caused bugs in the past.

**Review by abalmos — CHANGES_REQUESTED** [review]
Left a few comments.

Also, I think the back button on `/fields/[id]` should go back to map if we came from the map and back to list if coming from the list.

Maybe we need a better way to deal with back button long term.

**Review by KennethWrong — COMMENTED** [review]
ok I have made the name of the function more descriptive

File: yarn.lock (hunk 4 above)
Comment (KennethWrong) [reply yarn.lock]: I switched to a previous commit before updating svelte kit and all that and redid the function. Please check to see if they all work out.

**Comment by abalmos** [comment]
Thanks @KennethWrong 

I'll merge, I also prefer snake case but camelCase is well established in JS/TS world.

This still lacks the back button fix, but we need a `gotoX` libraryhat knows how to properly deal with the back button. We can fix the case then too.

**Review by abalmos — APPROVED** [review]

**Pull request merged**